compiler/rustc_expand/src/base.rs

```diff
@@ -590,6 +591,10 @@ impl MacResult for DummyResult {
         Some(P(DummyResult::raw_pat(self.span)))
     }
 
+    fn make_crate(self: Box<DummyResult>) -> Option<ast::Crate> {
+        Some(DummyAstNode::dummy())
+    }
```

Comment:
Nit: could you keep the order of methods in the impl and in the trait consistent?